app/zipzip-api-app/src/main/java/org/ssafy/zipzipapiapp/workspace/service/WorkspaceService.java

```diff
@@ -35,4 +38,27 @@ public void post(PostWorkspaceRequest postWorkspaceRequest, Long memberId) {
         workspaceMemberRepository.save(workspaceMember);
     }
 
+    @Transactional
+    public void patch(PatchWorkspaceRequest patchWorkspaceRequest, Long workspaceId) {
+        List<Long> memberIdList = patchWorkspaceRequest.memberIdList();
+        String workspaceName = patchWorkspaceRequest.workspaceName();
+
+        workspaceRepository.update(workspaceName, workspaceId);
+
+        workspaceMemberRepository.deleteAllByWorkspaceIdExceptOwner(workspaceId);
+
+        List<WorkspaceMember> workspaceMemberList = new ArrayList<>();
+        for (Long memberId : memberIdList) {
+            workspaceMemberList.add(WorkspaceMember.builder()
+                    .workspaceId(workspaceId)
+                    .memberId(memberId)
+                    .memberRole(WorkspaceMemberRole.MEMBER)
+                    .build()
+            );
+        }
```

Comment:
stream 써서 간단하게 나타내는 것도 좋을 것 같아요 !

```suggestion

List<WorkspaceMember> workspaceMemberList = memberIdList.stream()
        .map(memberId -> WorkspaceMember.builder()
                .workspaceId(workspaceId)
                .memberId(memberId)
                .memberRole(WorkspaceMemberRole.MEMBER)
                .build())
        .collect(Collectors.toList());

```

Reply:
넵 변경하도록 하겠습니다~